### PR TITLE
Define defaults role values

### PR DESCRIPTION
**Pull request by debo**

### CODE REVIEW

File: tasks/mod-php.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 - name: ensure timezone is set in apache2 php.ini
   lineinfile: dest=/etc/php5/apache2/php.ini
               regexp='date.timezone ='
-              line='date.timezone = {{ server.timezone }}'
+              line='date.timezone = {{ php.timezone }}'
```
Comment (naxhh) [inline tasks/mod-php.yml]: For now don't change this... just ad a server.timezone in the defaults....

I know.. not cute but I want to make the change in phansible transparent.
And this kind of changes will require modifying the options in phansible..

Create an issue for it and we will take care when phansible starts using this roles.

Comment (debo) [inline tasks/mod-php.yml]: Ok sure, no problem.

**Comment by debo** [comment]
@naxhh done.

**Comment by naxhh** [comment]
Thanks! This weekend I've to do a trip.
Let's see if I get time to help you here :)

**Comment by debo** [comment]
Hehe don't worry, I'm having a lot of fun. Thanks though

**Pull request merged**